case-studies/bayesline.md

```diff
@@ -1,6 +1,6 @@
 ---
 company: Bayesline
-description: "How Bayesline builds 4x faster using Reflex instead of Plotly Dash"
+description: "How Bayesline built a production grade web app in pure Python with 50% less code using Reflex instead of Plotly Dash"
```

Comment:
Can we shorten this up its a little wordy - Why Basyesline Chose Reflex over Dash for their flagship applications

Reply:
edited it to make it shorter, but need to keep python and web app in it as this the thing that will seo